app/src/main/res/layout/concept_card_fragment.xml

```diff
@@ -1,50 +1,87 @@
+      app:title="@string/concept_card_title"
+      app:titleTextColor="@color/white" />
+
+    <View
+      android:id="@+id/toolbar_shadow_view"
+      android:layout_width="match_parent"
+      android:layout_height="6dp"
+      android:background="@drawable/toolbar_drop_shadow"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/concept_card_toolbar" />
+
     <ScrollView
       android:layout_width="match_parent"
       android:layout_height="0dp"
-      app:layout_constraintTop_toBottomOf="@+id/concept_card_toolbar"
+      app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintLeft_toLeftOf="parent"
       app:layout_constraintRight_toRightOf="parent"
-      app:layout_constraintBottom_toBottomOf="parent">
+      app:layout_constraintTop_toBottomOf="@+id/toolbar_shadow_view">
+
       <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:paddingTop="10dp">
+        android:orientation="vertical">
+
         <TextView
-          android:id="@+id/explanation"
-          android:layout_width="match_parent"
+          android:id="@+id/description_text_view"
+          android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:textColor="@android:color/black"
-          android:gravity="center"
-          android:visibility="@{viewModel.conceptCardLiveData.hasExplanation() ? View.VISIBLE : View.GONE}"
-          android:text="@{viewModel.conceptCardLiveData.explanation.getHtml()}" />
-        <androidx.recyclerview.widget.RecyclerView
-          android:id="@+id/worked_examples"
+          android:layout_marginStart="28dp"
+          android:layout_marginTop="40dp"
+          android:layout_marginEnd="28dp"
+          android:fontFamily="sans-serif-medium"
+          android:text="@{viewModel.conceptCard.get().skillDescription}"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="20sp" />
+
+        <TextView
+          android:id="@+id/explanation_text_view"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
-          app:data="@{viewModel.workedExamplesLiveData}" />
+          android:layout_marginStart="40dp"
+          android:layout_marginTop="28dp"
+          android:layout_marginEnd="40dp"
+          android:fontFamily="sans-serif"
+          android:text="@{parsedExplanation}"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="16sp"
+          android:visibility="@{viewModel.conceptCard.get().hasExplanation() ? View.VISIBLE : View.GONE}" />
+        <!--TODO(#352): Show worked examples in Concept Card.-->
+        <!--<androidx.recyclerview.widget.RecyclerView-->
```

Comment:
Please remove rather than comment out. We should still keep the TODO, but let’s not check in commented out code.

Reply:
Done. Removed commented code and kept the TODO.